functions/controllers/expressExample/index.js

```diff
@@ -26,7 +29,47 @@ app.get("/", async (req, res) => {
 });
 
 app.post("/resend", jwtCheck, async (req, res) => {
-  res.status(201).send({ message: "private body reached" });
+  // this endpoint takes in a user_id then converts
+  // it into an API call
+  try {
```

Comment:
Can you pull the token with our DB and check in with the @rvaidun to make sure you are able to have it functioning via retrieving and have our tests updated for this as well.